qiskit/compiler/assembler.py

```diff
@@ -126,3 +132,151 @@ def assemble_circuits(circuits, run_config=None, qobj_header=None, qobj_id=None)
 
     return QasmQobj(qobj_id=qobj_id or str(uuid.uuid4()), config=userconfig,
                     experiments=experiments, header=qobj_header)
+
+
+def assemble_schedules(schedules, dict_config, dict_header):
+    """Assembles a list of circuits into a qobj which can be run on the backend.
+
+    Args:
+        schedules (list[ConfiguredSchedule] or ConfiguredSchedule): schedules to assemble
+        dict_config (dict): configuration of experiments
+        dict_header (dict): header to pass to the results
+
+    Returns:
+        PulseQobj: the Qobj to be run on the backends
+
+    Raises:
+        QiskitError: when invalid command is provided
+    """
+    if isinstance(schedules, ConfiguredSchedule):
+        schedules = [schedules]
+
+    experiments = []
+
+    default_qubit_lo_freq = dict_config.get('qubit_lo_freq', None)
+    default_meas_lo_freq = dict_config.get('meas_lo_freq', None)
+
+    user_pulselib = set()
+    for ii, configed in enumerate(schedules):
```

Comment:
Could you please use a more explicit variable name than `ii`, is this something like `exp_idx`? Also, what is the meaning of `configed`? Shorthand for `configured`?